### ISSUE TITLE
Expression issue of BatchNormalization layer

### ISSUE THREAD

**Issue by zwang1995** [tensorflow/tensorflow#48039]
A issue about the expression in the BatchNormalization layer.

## URL(s) with the issue:
https://keras.io/api/layers/normalization_layers/batch_normalization/

## Description of issue (what needs changing):
In the provided URL, during inference, this layer returns:
(batch - self.moving_mean) / (self.moving_var + epsilon) * gamma + beta.

But, I think a square operation is missing. It should return:
(batch - self.moving_mean) / **sqrt**(self.moving_var + epsilon) * gamma + beta.

Similarly, the square operation should be add in the equation during training, as:
(batch - mean(batch)) / **sqrt**(var(batch) + epsilon) * gamma + beta.

Please refer to Algorithm 1 in the paper: 
_Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift_

**Comment by rmothukuru** [comment]
@zwang1995,
Can you please share the link corresponding to the statement, 

> Please refer to Algorithm 1 in the paper:
> Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift.

so that we can double check? Thanks!

**Comment by zwang1995** [comment]
@rmothukuru 
Here is the link: https://arxiv.org/abs/1502.03167

**Comment by rmothukuru** [comment]
@zwang1995,
Sorry, but we couldn't find the expression you mentioned in that link. Can you please provide the exact link? Thanks!

**Comment by zwang1995** [comment]
@rmothukuru 
In Algorithm 1, in the normalize step, there is a **square root** for (sigma**2 + epsilon). 
And the **square root** is missing in the document for BatchNormalization layer.

**Comment by rmothukuru** [comment]
@zwang1995,
If I am not wrong, you are referring to the `formulae` mentioned in [this link](https://paperswithcode.com/method/batch-normalization). Your point is valid but the expression has been simplified by removing the `Square Root` in the `Denominator` because **_`Square root of (sigma ** 2)`_** will be **_`sigma`_** itself, and the value, **_`epsilon`_** is a very very small number (usually negligible), which has been added to avoid **_`Divide By Zero`_** condition.

Hope this clarifies about the expressions in the [documentation of Batch Normalization](https://www.tensorflow.org/api_docs/python/tf/keras/layers/BatchNormalization). Thanks!

**Comment by zwang1995** [comment]
@rmothukuru,
It did clarified. Thank you!
